primefaces/src/main/resources/META-INF/resources/primefaces/captcha/captcha.js

```diff
@@ -31,6 +31,7 @@ PrimeFaces.widget.Captcha = PrimeFaces.widget.BaseWidget.extend({
     init: function(cfg) {
         this._super(cfg);
         this.cfg.language = this.cfg.language||'en';
+        this.cfg.theme = this.cfg.theme || PrimeFaces.env.getPreferredColorScheme();
```

Comment:
This would work in combination with `pe:lightSwitch`, but is was planning to consider the current theme to make this more bulletproof. I think you can do theme is Arya, Vela or ends with "-dark" or "-dim". See for example https://primefaces.org/diamond/javax.faces.resource/theme.css.xhtml?ln=primefaces-diamond-blue-dark

Reply:
Ahh good idea.